.ort.yml

```diff
@@ -1,15 +1,15 @@
 ---
 excludes:
   paths:
-  - pattern: "**/src/{funTest,test}/**"
+  - pattern: "**/src/{:funTest|test}/**"
```

Comment:
Just as a side note: IMO that new syntax is less intuitive, and it's a bit sad we have to do this, but it's good to have this aligned and documented now; maybe we find a way to revert to the previous syntax by using (yet) another glob matcher implementation in the future...